openapi.yml

```diff
@@ -57,6 +57,31 @@ paths:
           required: false
           schema:
             type: integer
+      requestBody:
+        required: false
+        content:
+          '*/*':
```

Comment:
i found that i needed this section for backwards compatibility with the existing `/dor/reindex` usage

Reply:
@jmartin-sul and I chatted about this and we think it's worth seeing if flipping (Committee gem config) `parse_response_by_content_type` to `false` (in `config/application.rb`) can suffice to allow the old behavior without allowing all content types for an optional body.